bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -150,65 +148,87 @@ public static String executeUrl(String httpMethod, String url, Properties httpHe
             String contentType, int timeout, String proxyHost, Integer proxyPort, String proxyUser,
             String proxyPassword, String nonProxyHosts) {
 
-        HttpClient client = new HttpClient();
+        startHttpClient(client);
 
         // only configure a proxy if a host is provided
         if (StringUtils.isNotBlank(proxyHost) && proxyPort != null && shouldUseProxy(url, nonProxyHosts)) {
-            client.getHostConfiguration().setProxy(proxyHost, proxyPort);
-            if (StringUtils.isNotBlank(proxyUser)) {
-                client.getState().setProxyCredentials(AuthScope.ANY,
-                        new UsernamePasswordCredentials(proxyUser, proxyPassword));
+            AuthenticationStore authStore = client.getAuthenticationStore();
+            ProxyConfiguration proxyConfig = client.getProxyConfiguration();
+
+            // clear proxy list from previous requests
+            List<Proxy> proxies = proxyConfig.getProxies();
+            if (!proxies.isEmpty()) {
+                proxies.clear();
```

Comment:
As we are using the same store for possibly concurrent requests, you must not clear all proxies here.
Instead you should remove the proxy that you have added for this request after the request is done - avoid any side effects on concurrently running requests!